uber/templates/groupform.html

```diff
@@ -32,8 +32,15 @@
     {% endif %}
 {% else %}
     <script>
+        var showOrHideCategoriesText = function() {
+            setVisible('#category-other', $(":checkbox[name=categories][value={{ c.OTHER }}]").prop("checked"));
+        };
+
         $(function () {
             $('#bold-field-message').insertBefore($.field('name').parents('.form-group'));
+            $(":checkbox[name=categories][value={{ c.OTHER }}]").parents("label").append('<input id="category-other" class="form-control" type="text" name="categories_text" value="{{ group.categories_text }}" />');
```

Comment:
Hmm. I put it where it was because I wanted the input box to be 'attached' to the Other checkbox. Now that I see it though, I think it'll be okay if it's got that placeholder text that explains what it's for. I'll make that change!